lib/Dialect/CKKS/IR/CKKSOps.td

```diff
@@ -85,7 +85,8 @@ def CKKS_MulOp : CKKS_Op<"mul", [Commutative, SameOperandsAndResultRings, InferT
   let hasVerifier = 1;
 }
 
-def CKKS_MulPlainOp : CKKS_CiphertextPlaintextOp<"mul_plain", [InferTypeOpAdaptor, AllCiphertextTypesMatch, Commutative]> {
+// MulPlain op result ciphertext type could be different from the input
+def CKKS_MulPlainOp : CKKS_CiphertextPlaintextOp<"mul_plain", [InferTypeOpAdaptor, Commutative]> {
```

Comment:
Cc @asraa on #1620: BGV/CKKS MulPlainOp does not require result and input ciphertext types to be the same in that they may have different scale. The verification of result scale = ct scale + pt scale is done by InferTypeOpAdaptor in https://github.com/google/heir/blob/55d883daa628be169db3f8c0e05b1076e9b3328d/lib/Dialect/LWE/IR/LWEOps.h#L69-L78